flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/backends/hdfs/HDFSBackendImpl.java

```diff
@@ -237,4 +258,95 @@ private HttpResponse doHDFSRequest(String method, String url, ArrayList<Header>
         return response;
     } // doHDFSRequest
     
+    // from here on, consider this link:
+    // http://stackoverflow.com/questions/21629132/httpclient-set-credentials-for-kerberos-authentication
+    private HttpResponse doPrivilegedHDFSRequest(String method, String url, ArrayList<Header> headers,
+            StringEntity entity) throws Exception {
+        try {
+            LoginContext loginContext = new LoginContext("cygnus_krb5_login",
```

Comment:
In that case, probably is a good idea to add in krb5_login.conf just above the `cygnus_krb5_login {` a big disclaimer:

"DON NOT CHANGE THE cygnus_krb5_login NAME!! This name is used internally in Cygnus code, so doing so, will break the program"

or something like that.